runtime/src/utils/cell.rs

```diff
@@ -63,26 +63,36 @@ impl<C> Cell<C> {
         }
     }
 
-    /// Consume the slot, returning the context and panicking if it is missing.
-    pub fn into(self) -> C {
+    /// Returns a reference to the context.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the context is missing.
+    pub fn as_present(&self) -> &C {
```

Comment:
Is `as_present*` common lingo for this (just curious)?

Reply:
`as_present` isn't, just using `as_<variant_name>`. Generally borrow -> borrow conversions are `as_` though: https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv